nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -3180,14 +3284,14 @@ In this example, Nginx is used as a reverse proxy.
 
 image:s2s-rproxy-servername.svg["Server name to Node mapping"]
 
-1. Client1 initiates Site-to-Site protocol, the request is routed to one of upstream NiFi nodes. The NiFi node computes Site-to-Site port for RAW. By the routing rule 'example1' in nifi.properties shown below, port 10443 is returned.
+1. Client1 initiates Site-to-Site protocol, the request is routed to one of upstream NiFi nodes. The NiFi node computes Site-to-Site port for RAW. By the routing rule 'example1' in _nifi.properties_ shown below, port 10443 is returned.
```

Comment:
All single quote delimited (`'`) values in the next 6 lines should be `code formatted`.